content/docs/latest/provisioning/config-transpiler/configuration.md

```diff
@@ -45,7 +45,7 @@ The Butane Flatcar variant configuration is a YAML document conforming to the fo
         * **_verification_** (object): options related to the verification of the certificate.
           * **_hash_** (string): the hash of the certificate, in the form `<type>-<value>` where type is either `sha512` or `sha256`.
   * **_proxy_** (object): options relating to setting an `HTTP(S)` proxy when fetching resources.
-    * **_http_proxy_** (string): will be used as the proxy URL for HTTP requests and HTTPS requests unless overridden by `https_proxy` or `no_proxy`.
+    * **_http_proxy_** (string): will be used as the proxy URL for HTTP requests unless overridden by `no_proxy`.
```

Comment:
Thanks. The Flatcar docs copy of the Ignition config spec also needs to be fixed.

Also, we should do the same change upstream in the Ignition specs and Butane specs (linked specific files here but there's a copy for each variant/version):
https://github.com/coreos/ignition/blob/main/docs/configuration-v3_6_experimental.md
https://github.com/coreos/butane/blob/main/docs/config-flatcar-v1_2-exp.md